acme/src/acme/client.py

```diff
@@ -314,44 +314,43 @@ def finalize_order(self, orderr: messages.OrderResource, deadline: datetime.date
                 raise e
         return self.poll_finalization(orderr, deadline, fetch_alternative_chains)
 
-    def renewal_time(self, cert_pem: bytes) -> Tuple[datetime.datetime, datetime.datetime]:
+    def renewal_time(self, cert_pem: bytes
+        ) -> Tuple[Optional[datetime.datetime], datetime.datetime]:
         """Return an appropriate time to attempt renewal of the certificate,
         and the next time to ask the ACME server for renewal info.
 
         If the ACME directory has a "renewalInfo" field, the response will be
         based on a fetch of the renewal info resource for the certificate
         (https://www.ietf.org/archive/id/draft-ietf-acme-ari-08.html).
 
-        If there is no "renewalInfo" field, this function will fall back to
-        reasonable defaults based on the certificate lifetime.
+        If there is no "renewalInfo" field, this function will return a tuple of
+        None, and the next time to ask the ACME server for renewal info.
 
         This function may make other network calls in the future (e.g., OCSP
         or CRL).
+
+        :param bytes cert_pem: cert as pem file
+
+        :returns: Tuple of time to attempt renewal, next time to ask for renewal info
+        :rtype: (Optional[`datetime.datetime`], `datetime.datetime`)
```

Comment:
oh python. why do we have to document parameter and return types in two places using different syntax?

i'm not sure sphinx understands the line i'm commenting on here. the generated documentation looks like:
![Screenshot 2025-06-04 at 9 52 03 AM](https://github.com/user-attachments/assets/ed0480f0-ba51-4289-9fdd-cc9724bbf3fc)

maybe instead we could do
```suggestion
        :rtype: `tuple` of two `datetime.datetime`, first may be `None`
```
which is kind of awkward, but i think better matches the approach currently taken in our codebase and looks like
![Screenshot 2025-06-04 at 9 57 24 AM](https://github.com/user-attachments/assets/facd9530-c54c-4703-8a09-ac527e4cb6ed)

alternatively, maybe we could just delete this line since the return type is already documented in the function signature?
```suggestion
```
that looks how you'd expect with just no return type field at the bottom of the documentation

for better or worse, i nerded out about this a bit and wrote up https://github.com/certbot/certbot/issues/10310 suggesting we look into ways to stop documenting type information in two different places